packages/antd/src/widgets/CheckboxWidget/index.tsx

```diff
@@ -34,19 +57,33 @@ export default function CheckboxWidget<
     onBlur: !readonly ? handleBlur : undefined,
     onFocus: !readonly ? handleFocus : undefined,
   };
+  const description = options.description ?? schema.description;
 
   return (
-    <Checkbox
-      autoFocus={autofocus}
-      checked={typeof value === 'undefined' ? false : value}
-      disabled={disabled || (readonlyAsDisabled && readonly)}
-      id={id}
-      name={id}
-      onChange={!readonly ? handleChange : undefined}
-      {...extraProps}
-      aria-describedby={ariaDescribedByIds<T>(id)}
-    >
-      {label}
-    </Checkbox>
+    <>
+      {displayLabel && !!description && (
+        <div>
```

Comment:
The antd theme's description is a span, wrapped it in a div so that the description is in a block element